### ISSUE TITLE
[Feature Request] Rememember table layout and sorting

### ISSUE THREAD

**Issue by poperigby** [limo-app/limo#45]
Every time I restart LIMO, it forgets that I've adjusted the width of the mods table, and sorted them by ID.

**Comment by limo-app** [comment]
I agree that the sort order should be remembered. This will probably be part of 1.0.9.

Currently column widths are auto-adjusted every time the table is update to adapt to the potential change of maximum mod name length. I believe adding an option to instead remember column sizes would be best in this case.
